### ISSUE TITLE
UNIX_OFFSET not defined for ESP32

### ISSUE THREAD

**Issue by sean1587** [cvmanjoo/RTC#17]
I can't compile on ESP32.

My work around is to change code in I2C_RTC.h from

```
#if defined(ARDUINO_ARCH_RP2040)
#define UNIX_OFFSET 946684800
#endif
```
to 
`#define UNIX_OFFSET 946684800`

Can you make the library compatible to the ESP32 please, so thats no need to manualy change library code?

**Comment by gogreenpower** [comment]
second this

**Comment by cvmanjoo** [comment]
Fixed this in v1.7.1
